app/src/main/java/ca/marshallasch/veil/PostListAdapter.java

```diff
@@ -190,6 +196,55 @@ public void sort(SortOption option) {
      */
     @Override
     public int getItemCount() {
-        return posts.size();
+        return postsFiltered.size();
+    }
+
+
+    @Override
+    public Filter getFilter()
+    {
+
+        /**
+         * This class is the tag filter. The CharSequence must be a list of tags that are
+         * denominated with a ':' character.
```

Comment:
Maybe use another character instead of ':' since its common to make a post like so "Sample Title: somthing somthing".